LLHInterface.h

```diff
@@ -21,6 +21,26 @@ Bifrost& operator<<(Bifrost& bf, OscPars& p)
 void SendLLHOverBifrost(Bifrost& bf, TemplateLLHGetter* llh)
 {
   while(true){
+    //send asimov osc pars
+    OscPars asimovOscPars;
+    int gotSentAsimovOsc;
+
+    bf >> gotSentAsimovOsc;
+
+    if(gotSentAsimovOsc) {
+      bf >> asimovOscPars;
+      llh->SetAsimovOscParameters(asimovOscPars);
+    }
```

Comment:
Move the OscPars to the scope they are actually needed
```suggestion
    int gotSentAsimovOsc;

    bf >> gotSentAsimovOsc;

    if(gotSentAsimovOsc) {
      OscPars asimovOscPars;
      bf >> asimovOscPars;
      llh->SetAsimovOscParameters(asimovOscPars);
    }
```